jgayfer/event_bot
# Setup event response with reactions

This PR adds functionality for users to indicate a response to an event (accepted, declined, or alternate). When a valid reaction is added, the database is updated with the updated response.

Aside from the reactions themselves, the changes are only in the back end at this point; the event is not yet updated to indicate the user's response.

#### To verify this change:

1. Create an event
2. Click on the reactions that are added to the event (multiple times)
3. Observe that the database is updated with new attendance value (shown in console)
